Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -21,6 +21,43 @@ public static Command SelectActionMap(string actionMapName)
             return (in InputActionsEditorState state) => state.SelectActionMap(actionMapName);
         }
 
+        public static Command AddActionMap()
+        {
+            return (in InputActionsEditorState state) =>
+            {
+                var newMap = InputActionSerializationHelpers.AddActionMap(state.serializedObject);
+                var actionProperty = InputActionSerializationHelpers.AddAction(newMap);
+                InputActionSerializationHelpers.AddBinding(actionProperty, newMap);
```

Comment:
Why do we add a action and binding alongside the action map ?
Does it have to have at least one to be valid?

Reply:
Unfortunately we have to, it seems we are trying to access the binding array (or actions array) and when we don't have bindings (or actions) we will get an element out of bounds exception. It was the same way with the old "non-toolkit" editor window, that's why we added one action and one binding there by default as well.

Reply:
I think it this is parity with the UGUI version then this is fine. Thanks for confirming